frontend/public/components/monitoring/alerting.tsx

```diff
@@ -212,14 +232,30 @@ const SeverityIcon: React.FC<{ severity: string }> = ({ severity }) => {
   return <Icon />;
 };
 
-export const Severity: React.FC<{ severity: string }> = ({ severity }) =>
-  _.isNil(severity) ? (
+export const Severity: React.FC<{ severity: string }> = ({ severity }) => {
+  const { t } = useTranslation();
+
+  const getSeverityKey = (severityData) => {
```

Comment:
Missing type. Is this really `severityData` or simply a `severity`? :)

Reply:
`severity` is already defined, so I had to rename it. Open to suggestions.